File: protoqueue-runtime/src/main/java/net/protoqueue/ProtoError.java
package net.protoqueue;

/**
 * Created by zhongyongsheng on 2018/5/6.
 */
public class ProtoError extends Exception{

    public ProtoError() {
    }

    public ProtoError(String message) {
        super(message);
    }

    public ProtoError(String message, Throwable cause) {
        super(message, cause);
    }
}

File: protoqueue-runtime/src/main/java/net/protoqueue/ProtoError.kt
package net.protoqueue

/**
 * ProtoQueue异常类型
 * Created by zhongyongsheng on 2018/5/6.
 */
open class ProtoError : Exception {
    constructor() {}
    constructor(message: String?) : super(message) {}
    constructor(message: String?, cause: Throwable?) : super(message, cause) {}
}